js/fullWidthTreeView.js

```diff
@@ -224,6 +224,22 @@
 
         // Update the url, TODO save the state
         window.history.pushState(null, null, url);
+
+        // Get new document title for G4 Tracking
+        var doctitle = $('#main-column h1').first();
+
+        // Create a temporary element to parse the HTML
+        var tempElement = document.createElement('div');
+        tempElement.innerHTML = doctitle[0].outerHTML;
+
+        // Extract the text content without HTML tags and set title
+        document.title = tempElement.textContent;
```

Comment:
You should be able to get the document title without creating an extra div element. `doctitle[0].textContent` should give you the title I believe.